### DISCUSSION TITLE
Show the name of countries in other languages than default when using i18n translation system

### DISCUSSION THREAD

**Discussion by Rahimz** [SmileyChris/django-countries#379] [Q&A] [answered]
I have installed the django_contries and it works great with Rosetta translation package but in some templates I need to use the English name of country for example when I want to show an international version of an address, but Rosetta translates the country name to my default i18n language.
Is there any way to stop translation of countries name in some cases(Like international address) ?  in other part of my app I need the translated names.

**Comment by SmileyChris** [comment, marked as answer]
Sounds like this is more of a standard django translation issue than a django-countries one.

In this case specifically, probably ensuring you're in english when you're rendering the data (or the form, if that's where you're wanting it in english)
```django
{% language 'en' %}
    <p>{{ person.country }}</p>
{% endlanguage %}
```
See Django's docs on [switching language in templates](https://docs.djangoproject.com/en/3.2/topics/i18n/translation/#switching-language-in-templates).
